src/com/google/javascript/jscomp/PolymerClassDefinition.java

```diff
@@ -33,6 +33,8 @@
  * class.
  */
 final class PolymerClassDefinition {
+  /** Either 1 or 2 - the Polymer version of the definition */
+  final int version;
```

Comment:
I can change the open source flags to indicate this, but you'll have to handle the internal version.

That change will take me a bit longer than the rest (which will be pushed in just a few minutes).